crates/node/src/lib.rs

```diff
@@ -242,7 +240,9 @@ async fn handle_line(node: &mut Node, line: String) -> eyre::Result<()> {
                     }
                 }
             } else {
-                println!("{IND} Usage: call <Method> <JSON Payload>");
+                println!(
+                    "{IND} Usage: call <Context ID> <Method> <JSON Payload> [Executor Public Key]"
```

Comment:
not exactly optional

```suggestion
                    "{IND} Usage: call <Context ID> <Method> <JSON Payload> <Executor Public Key>"
```